src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs

```diff
@@ -152,56 +152,6 @@ public bool RemoveSuppressions(bool selectedErrorListEntriesOnly, IVsHierarchy?
         return p => projectHierarchy == null || p.Id == projectIdToMatch;
     }
 
-    private async Task<ImmutableArray<DiagnosticData>> GetAllBuildDiagnosticsAsync(Func<Project, bool> shouldFixInProject, CancellationToken cancellationToken)
-    {
-        using var _ = CodeAnalysis.PooledObjects.ArrayBuilder<DiagnosticData>.GetInstance(out var builder);
-
-        var buildDiagnostics = _buildErrorDiagnosticService.GetBuildErrors().Where(d => d.ProjectId != null && d.Severity != DiagnosticSeverity.Hidden);
```

Comment:
This never returned diagnostics with LSP on.  per convo with @CyrusNajmabadi that is fine.